src/test/java/com/epam/tc/hw2/ex1/FirstExercise.java

```diff
@@ -0,0 +1,135 @@
+        softly.assertThat(isContactFormImagePresent).isTrue();
+
+        String actualServiceData = driver.findElement(SERVICE_BUTTON).getText();
+        softly.assertThat(actualServiceData).isEqualTo(EXPECTED_SERVICE_DATA);
+
+        boolean isServiceImagePresent = driver.findElement(SERVICE_BUTTON).isDisplayed();
+        softly.assertThat(isServiceImagePresent).isTrue();
+
+        String actualMetalsAndColorsData = driver.findElement(METALS_AND_COLORS_BUTTON).getText();
+        softly.assertThat(actualMetalsAndColorsData).isEqualTo(EXPECTED_METALS_AND_COLORS_DATA);
+
+        boolean isMetalsAndColorsImagePresent = driver.findElement(METALS_AND_COLORS_BUTTON).isDisplayed();
+        softly.assertThat(isMetalsAndColorsImagePresent).isTrue();
+
+        //6. Assert that there are 4 images on the Index Page and they are displayed
+        WebElement firstImage = driver.findElement(FIRST_ICON);
+        boolean firstImagePresent = firstImage.isDisplayed();
+        softly.assertThat(firstImagePresent).isTrue();
+
+        boolean isSecondImagePresent = driver.findElement(SECOND_ICON).isDisplayed();
+        softly.assertThat(isSecondImagePresent).isTrue();
+
+        boolean isThirdImagePresent = driver.findElement(THIRD_ICON).isDisplayed();
+        softly.assertThat(isThirdImagePresent).isTrue();
+
+        boolean fourthImagePresent = driver.findElement(FOURTH_ICON).isDisplayed();
+        softly.assertThat(fourthImagePresent).isTrue();
+
+        //7. Assert that there are 4 texts on the Index Page under icons and they have proper text
+        String actualTextUnderFirstImage = driver.findElement(TEXT_UNDER_FIRST_IMAGE).getText();
+        softly.assertThat(actualTextUnderFirstImage).isEqualTo(EXPECTED_TEXT_UNDER_FIRST_IMAGE);
+
+        String actualTextUnderSecondImage = driver.findElement(TEXT_UNDER_SECOND_IMAGE).getText();
+        softly.assertThat(actualTextUnderSecondImage).isEqualTo(EXPECTED_TEXT_UNDER_SECOND_IMAGE);
+
+        String actualTextUnderThirdImage = driver.findElement(TEXT_UNDER_THIRD_IMAGE).getText();
+        softly.assertThat(actualTextUnderThirdImage).isEqualTo(EXPECTED_TEXT_UNDER_THIRD_IMAGE);
+
+        String actualTextUnderFourthImage = driver.findElement(TEXT_UNDER_FOURTH_IMAGE).getText();
+        softly.assertThat(actualTextUnderFourthImage).isEqualTo(EXPECTED_TEXT_UNDER_FOURTH_IMAGE);
+
+        //8. Assert that there is the iframe with “Frame Button” exist
+        boolean isActualIframeExists = (new WebDriverWait(driver, 5))
+                .until(ExpectedConditions.invisibilityOfElementWithText(IFRAME, "The iframe exists"));
+        softly.assertThat(isActualIframeExists).isTrue();
+
+        //9. Switch to the iframe and check that there is “Frame Button” in the iframe
+        WebElement iframeElement = driver.findElement(IFRAME);
+        driver.switchTo().frame(iframeElement);
+
+        boolean isFrameButtonElementPresent = (new WebDriverWait(driver, 5))
```

Comment:
brackets is extra here.
`WebDriverWait` could be specified in the start of the tests

Reply:
Now WebDriverWait is specified in the start of the tests in TestService class.